api/v1/helper.go

```diff
@@ -271,6 +279,28 @@ func NeedToUpdateSriov(ifaceSpec *Interface, ifaceStatus *InterfaceExt) bool {
 							return true
 						}
 
+						if strings.EqualFold(ifaceStatus.LinkType, consts.LinkTypeETH) {
+							if len(vfStatus.Mac) == 0 || vfStatus.Mac == consts.DefaultMAC {
```

Comment:
I have been thinking on some possible scenarios which might make checking  mac/guid tricky.

for MAC: if VF netdev is used by a container then you will not be able to discover its netdev hence its mac will be `""`
which will trigger re-configuration.

for GUID basically the same but this will happen only when RDMA namespace mode is set to exclusive.

Reply:
we also could do some merge logic where if we fail to read a value, assume the previous one is still valid. but that also has its drawbacks.

Reply:
Makes sense. I will change the condition to just do checks based on default MAC and Node GUID.